### ISSUE TITLE
Javascript console Warning: Failed prop type: Invalid prop `valueRef` supplied to `DurationCondition`.

### ISSUE THREAD

**Issue by codefromthecrypt** [openzipkin/zipkin#3109]
If you click serviceName, then choose a duration the following console log happens

```
Warning: Failed prop type: Invalid prop `valueRef` supplied to `DurationCondition`.
    in DurationCondition (at GlobalSearchConditionValue.jsx:111)
    in GlobalSearchConditionValue (at GlobalSearchCondition.jsx:132)
    in div (created by ForwardRef(Paper))
    in ForwardRef(Paper) (created by WithStyles(ForwardRef(Paper)))
    in WithStyles(ForwardRef(Paper)) (at GlobalSearchCondition.jsx:124)
    in GlobalSearchCondition (at GlobalSearchConditionList.jsx:94)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at GlobalSearchConditionList.jsx:93)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at GlobalSearchConditionList.jsx:79)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at GlobalSearchConditionList.jsx:72)
    in GlobalSearchConditionList (at GlobalSearch.jsx:55)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at GlobalSearch.jsx:54)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at GlobalSearch.jsx:46)
    in GlobalSearch (created by Context.Consumer)
    in withRouter(GlobalSearch) (at DiscoverPage.jsx:272)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at DiscoverPage.jsx:270)
    in DiscoverPageContent (at DiscoverPage.jsx:323)
    in DiscoverPageImpl (created by Context.Consumer)
    in withRouter(DiscoverPageImpl) (created by ConnectFunction)
    in ConnectFunction (created by Context.Consumer)
    in Route (at App.jsx:51)
    in main (created by Styled(MuiBox))
    in Styled(MuiBox) (at Layout.jsx:132)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at Layout.jsx:85)
    in Layout (at App.jsx:50)
    in Router (created by BrowserRouter)
    in BrowserRouter (at App.jsx:49)
    in I18nProvider (at App.jsx:48)
    in Provider (at App.jsx:46)
    in ThemeProvider (at App.jsx:43)
    in MuiPickersUtilsProvider (at App.jsx:42)
    in UiConfig (at App.jsx:41)
    in Suspense (at App.jsx:40)
    in App (at src/index.tsx:21)
```

**Comment by codefromthecrypt** [comment]
same happens for a tag

```
index.js:1 Warning: Failed prop type: Invalid prop `valueRef` supplied to `TagCondition`.
    in TagCondition (at GlobalSearchConditionValue.jsx:113)
    in GlobalSearchConditionValue (at GlobalSearchCondition.jsx:132)
    in div (created by ForwardRef(Paper))
    in ForwardRef(Paper) (created by WithStyles(ForwardRef(Paper)))
    in WithStyles(ForwardRef(Paper)) (at GlobalSearchCondition.jsx:124)
    in GlobalSearchCondition (at GlobalSearchConditionList.jsx:94)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at GlobalSearchConditionList.jsx:93)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at GlobalSearchConditionList.jsx:79)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at GlobalSearchConditionList.jsx:72)
    in GlobalSearchConditionList (at GlobalSearch.jsx:55)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at GlobalSearch.jsx:54)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at GlobalSearch.jsx:46)
    in GlobalSearch (created by Context.Consumer)
    in withRouter(GlobalSearch) (at DiscoverPage.jsx:272)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at DiscoverPage.jsx:270)
    in DiscoverPageContent (at DiscoverPage.jsx:323)
    in DiscoverPageImpl (created by Context.Consumer)
    in withRouter(DiscoverPageImpl) (created by ConnectFunction)
    in ConnectFunction (created by Context.Consumer)
    in Route (at App.jsx:51)
    in main (created by Styled(MuiBox))
    in Styled(MuiBox) (at Layout.jsx:132)
    in div (created by Styled(MuiBox))
    in Styled(MuiBox) (at Layout.jsx:85)
    in Layout (at App.jsx:50)
    in Router (created by BrowserRouter)
    in BrowserRouter (at App.jsx:49)
    in I18nProvider (at App.jsx:48)
    in Provider (at App.jsx:46)
    in ThemeProvider (at App.jsx:43)
    in MuiPickersUtilsProvider (at App.jsx:42)
    in UiConfig (at App.jsx:41)
    in Suspense (at App.jsx:40)
    in App (at src/index.tsx:21)
```

**Comment by codefromthecrypt** [comment]
NOTE: this is only visible in dev mode (ex npm start)

**Comment by tacigar** [comment]
Maybe this issue is solved in #3076
